package.json

```diff
@@ -21,7 +21,7 @@
         "moment": "^2.24.0",
         "morgan": "~1.9.0",
         "nunjucks": "^3.1.7",
-        "q-transformer": "git+https://github.com/CriminalInjuriesCompensationAuthority/q-transformer.git"
+        "q-transformer": "git+https://github.com/CriminalInjuriesCompensationAuthority/q-transformer.git#govukselect-default-item"
```

Comment:
q-transformer is now up to date. Can you change this back (it'll have changed package.lock too which we don't want)

Reply:
Mind uninstall and reinstall so that package-lock.json gets upated